stumpy/stumped.py

```diff
@@ -385,6 +571,16 @@ def stumped(
     mparray([0.11633857, 2.69407392, 3.00009263, 2.69407392, 0.11633857])
     >>>         mp.I_
     mparray([4, 3, 0, 1, 0])
+
+    Alternatively, you can also use `ray`
+
+    >>> import ray
+    >>> if __name__ == "__main__":
+    >>>     ray.init()
+    >>>     stumpy.stumped(
+    ...             ray,
+    ...             np.array([584., -11., 23., 79., 1001., 0., -19.]),
+    ...             m=3)
```

Comment:
No, I don't think it's necessary as a minimum reproducible example. I don't really want to get into the logistics of using `ray`. In my opinion, it is the user's responsibility to figure out how to use `ray` (which involves spinning it up and shutting it down). This would be the same as for `dask` too. If `ray` had a context manager like `dask` then I would've done that